_rules/additional-content-triggered-on-hover-is-hoverable-ep1s13.md

```diff
@@ -0,0 +1,161 @@
+---
+id: ep1s13
+name: Additional content triggered on hover is hoverable
+rule_type: atomic
+description: |
+  This rule checks that any additional content that was triggered by hovering an element remains available to the user while the mouse pointer is over the element or the additional content
+accessibility_requirements:
+  wcag21:1.4.13: # Content on Hover or Focus (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Mouse pointer
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any element that when [hovered][] causes [visible changes in content][] and for the [area affected by the changes][area affected by visible changes] all the following are true:
+
+- the area is [adjacent][adjacent areas] or [overlaps][overlapping areas] the [bounding box][] of the target element; and
+- the area is not [contained][] inside the [bounding box][] of the target element.
+
+## Expectation
+
+There are no [visible changes in content][] in the [area affected by the changes][area affected by visible changes] while the target element is [hovered][] or the mouse pointer remains inside that area.
```

Comment:
I think this needs some kind of time component to ensure this is tested consistently. If I check this for a 5 seconds, and someone else tests it for a minute, we may get different results.

Reply:
I don't have nothing against it.